media/inf/shortcuts.inf

```diff
@@ -287,7 +287,7 @@ CALC_TITLE=Rechner
 CALC_DESC=Starte Taschenrechner
 NOTEPAD_TITLE=Editor
 NOTEPAD_DESC=Starte Texteditor
-MSTSC_TITLE=Remote Desktop
+MSTSC_TITLE=Remote Desktop Connection
```

Comment:
"Remote Desktop" is an acceptable translation, so either keep it or use Verbindung as found in the line below.

Reply:
Rest of the changes lgtm.

Reply:
> use Verbindung

Done.